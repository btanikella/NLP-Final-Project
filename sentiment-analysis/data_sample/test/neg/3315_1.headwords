before
say
mind
live
But
acceptable
's
laugh
happy
