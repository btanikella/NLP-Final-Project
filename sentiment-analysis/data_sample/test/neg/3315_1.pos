PRP VBP RB IN VBD CD IN DT NN IN DT NNP  CC IN DT CD PRP VBP JJ DT NN VBZ RB VB RB TO CD
DT PRP MD VB NN DT NN VBZ IN PRP VBD PRP IN DT NNP NNP  CC PRP VBD PRP$ JJS NN NN RB
PRP VBP RB VB DT JJ NNS
PRP MD VB IN DT JJ NN
CC WRB PRP VBP DT RB VBG  JJ NNS  IN PRP$ JJ NN -LRB- RB NNPS MD VB WP PRP VBP IN -RRB-  PRP VBD PRP NN IN NN PRP RB VBD PRP VBD JJ IN
-LRB- IN PRP VBD VBN PRP RP  DT NN MD RB VB VBN JJ : -RRB-
EX VBZ RB RB JJR TO VB VBN IN DT CD
PRP MD NN RB CC RB  IN PRP VBZ RB JJ  CC IN VBZ RB VB PRP DT JJR
PRP VBZ RB RB NN PRP VBP IN IN NNP  VB JJ CC VB IN DT JJ NN MD RBS JJ RB VBP PRP TO PRP$ NN PRP VBP VB IN NNP  VB DT NN IN DT NNS
