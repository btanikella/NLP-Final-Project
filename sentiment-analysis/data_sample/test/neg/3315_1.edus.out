Sentence #1 (29 tokens):
I have never before voted 1 for a movie on the IMDB, but for this one I am sorry the scale doesn't go down to -10.
[Text=I CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=PRP] [Text=have CharacterOffsetBegin=6 CharacterOffsetEnd=10 PartOfSpeech=VBP] [Text=never CharacterOffsetBegin=11 CharacterOffsetEnd=16 PartOfSpeech=RB] [Text=before CharacterOffsetBegin=17 CharacterOffsetEnd=23 PartOfSpeech=IN] [Text=voted CharacterOffsetBegin=24 CharacterOffsetEnd=29 PartOfSpeech=VBD] [Text=1 CharacterOffsetBegin=30 CharacterOffsetEnd=31 PartOfSpeech=CD] [Text=for CharacterOffsetBegin=32 CharacterOffsetEnd=35 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=36 CharacterOffsetEnd=37 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=38 CharacterOffsetEnd=43 PartOfSpeech=NN] [Text=on CharacterOffsetBegin=44 CharacterOffsetEnd=46 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=47 CharacterOffsetEnd=50 PartOfSpeech=DT] [Text=IMDB CharacterOffsetBegin=51 CharacterOffsetEnd=55 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=55 CharacterOffsetEnd=56 PartOfSpeech=,] [Text=but CharacterOffsetBegin=57 CharacterOffsetEnd=60 PartOfSpeech=CC] [Text=for CharacterOffsetBegin=61 CharacterOffsetEnd=64 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=65 CharacterOffsetEnd=69 PartOfSpeech=DT] [Text=one CharacterOffsetBegin=70 CharacterOffsetEnd=73 PartOfSpeech=CD] [Text=I CharacterOffsetBegin=74 CharacterOffsetEnd=75 PartOfSpeech=PRP] [Text=am CharacterOffsetBegin=76 CharacterOffsetEnd=78 PartOfSpeech=VBP] [Text=sorry CharacterOffsetBegin=79 CharacterOffsetEnd=84 PartOfSpeech=JJ] [Text=the CharacterOffsetBegin=85 CharacterOffsetEnd=88 PartOfSpeech=DT] [Text=scale CharacterOffsetBegin=89 CharacterOffsetEnd=94 PartOfSpeech=NN] [Text=does CharacterOffsetBegin=95 CharacterOffsetEnd=99 PartOfSpeech=VBZ] [Text=n't CharacterOffsetBegin=99 CharacterOffsetEnd=102 PartOfSpeech=RB] [Text=go CharacterOffsetBegin=103 CharacterOffsetEnd=105 PartOfSpeech=VB] [Text=down CharacterOffsetBegin=106 CharacterOffsetEnd=110 PartOfSpeech=RB] [Text=to CharacterOffsetBegin=111 CharacterOffsetEnd=113 PartOfSpeech=TO] [Text=-10 CharacterOffsetBegin=114 CharacterOffsetEnd=117 PartOfSpeech=CD] [Text=. CharacterOffsetBegin=117 CharacterOffsetEnd=118 PartOfSpeech=.] 
root(ROOT-0, before-4)
nsubj(before-4, I-1)
aux(before-4, have-2)
neg(before-4, never-3)
dep(before-4, voted-5)
dobj(voted-5, 1-6)
prep(voted-5, for-7)
det(movie-9, a-8)
pobj(for-7, movie-9)
prep(movie-9, on-10)
det(IMDB-12, the-11)
pobj(on-10, IMDB-12)
punct(voted-5, ,-13)
cc(voted-5, but-14)
conj(voted-5, for-15)
pobj(for-15, this-16)
amod(this-16, one-17)
nsubj(sorry-20, I-18)
cop(sorry-20, am-19)
rcmod(one-17, sorry-20)
det(scale-22, the-21)
nsubj(go-25, scale-22)
aux(go-25, does-23)
neg(go-25, n't-24)
ccomp(sorry-20, go-25)
advmod(go-25, down-26)
prep(go-25, to-27)
pobj(to-27, -10-28)
punct(before-4, .-29)

Sentence #2 (26 tokens):
All I can say abou this movie is that I saw it in a Sneak Preview, and it was my worst movie experience ever.
[Text=All CharacterOffsetBegin=118 CharacterOffsetEnd=121 PartOfSpeech=DT] [Text=I CharacterOffsetBegin=122 CharacterOffsetEnd=123 PartOfSpeech=PRP] [Text=can CharacterOffsetBegin=124 CharacterOffsetEnd=127 PartOfSpeech=MD] [Text=say CharacterOffsetBegin=128 CharacterOffsetEnd=131 PartOfSpeech=VB] [Text=abou CharacterOffsetBegin=132 CharacterOffsetEnd=136 PartOfSpeech=NN] [Text=this CharacterOffsetBegin=137 CharacterOffsetEnd=141 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=142 CharacterOffsetEnd=147 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=148 CharacterOffsetEnd=150 PartOfSpeech=VBZ] [Text=that CharacterOffsetBegin=151 CharacterOffsetEnd=155 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=156 CharacterOffsetEnd=157 PartOfSpeech=PRP] [Text=saw CharacterOffsetBegin=158 CharacterOffsetEnd=161 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=162 CharacterOffsetEnd=164 PartOfSpeech=PRP] [Text=in CharacterOffsetBegin=165 CharacterOffsetEnd=167 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=168 CharacterOffsetEnd=169 PartOfSpeech=DT] [Text=Sneak CharacterOffsetBegin=170 CharacterOffsetEnd=175 PartOfSpeech=NNP] [Text=Preview CharacterOffsetBegin=176 CharacterOffsetEnd=183 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=183 CharacterOffsetEnd=184 PartOfSpeech=,] [Text=and CharacterOffsetBegin=185 CharacterOffsetEnd=188 PartOfSpeech=CC] [Text=it CharacterOffsetBegin=189 CharacterOffsetEnd=191 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=192 CharacterOffsetEnd=195 PartOfSpeech=VBD] [Text=my CharacterOffsetBegin=196 CharacterOffsetEnd=198 PartOfSpeech=PRP$] [Text=worst CharacterOffsetBegin=199 CharacterOffsetEnd=204 PartOfSpeech=JJS] [Text=movie CharacterOffsetBegin=205 CharacterOffsetEnd=210 PartOfSpeech=NN] [Text=experience CharacterOffsetBegin=211 CharacterOffsetEnd=221 PartOfSpeech=NN] [Text=ever CharacterOffsetBegin=222 CharacterOffsetEnd=226 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=226 CharacterOffsetEnd=227 PartOfSpeech=.] 
root(ROOT-0, say-4)
dep(say-4, All-1)
nsubj(say-4, I-2)
aux(say-4, can-3)
nsubj(is-8, abou-5)
det(movie-7, this-6)
dobj(abou-5, movie-7)
ccomp(say-4, is-8)
mark(saw-11, that-9)
nsubj(saw-11, I-10)
ccomp(is-8, saw-11)
dobj(saw-11, it-12)
prep(saw-11, in-13)
det(Preview-16, a-14)
nn(Preview-16, Sneak-15)
pobj(in-13, Preview-16)
punct(saw-11, ,-17)
cc(saw-11, and-18)
nsubj(experience-24, it-19)
cop(experience-24, was-20)
poss(experience-24, my-21)
amod(experience-24, worst-22)
nn(experience-24, movie-23)
conj(saw-11, experience-24)
advmod(experience-24, ever-25)
punct(say-4, .-26)

Sentence #3 (8 tokens):
I don't mind the stupid jokes.
[Text=I CharacterOffsetBegin=232 CharacterOffsetEnd=233 PartOfSpeech=PRP] [Text=do CharacterOffsetBegin=234 CharacterOffsetEnd=236 PartOfSpeech=VBP] [Text=n't CharacterOffsetBegin=236 CharacterOffsetEnd=239 PartOfSpeech=RB] [Text=mind CharacterOffsetBegin=240 CharacterOffsetEnd=244 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=245 CharacterOffsetEnd=248 PartOfSpeech=DT] [Text=stupid CharacterOffsetBegin=249 CharacterOffsetEnd=255 PartOfSpeech=JJ] [Text=jokes CharacterOffsetBegin=256 CharacterOffsetEnd=261 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=261 CharacterOffsetEnd=262 PartOfSpeech=.] 
root(ROOT-0, mind-4)
nsubj(mind-4, I-1)
aux(mind-4, do-2)
neg(mind-4, n't-3)
det(jokes-7, the-5)
amod(jokes-7, stupid-6)
dobj(mind-4, jokes-7)
punct(mind-4, .-8)

Sentence #4 (8 tokens):
I can live with the silly story.
[Text=I CharacterOffsetBegin=267 CharacterOffsetEnd=268 PartOfSpeech=PRP] [Text=can CharacterOffsetBegin=269 CharacterOffsetEnd=272 PartOfSpeech=MD] [Text=live CharacterOffsetBegin=273 CharacterOffsetEnd=277 PartOfSpeech=VB] [Text=with CharacterOffsetBegin=278 CharacterOffsetEnd=282 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=283 CharacterOffsetEnd=286 PartOfSpeech=DT] [Text=silly CharacterOffsetBegin=287 CharacterOffsetEnd=292 PartOfSpeech=JJ] [Text=story CharacterOffsetBegin=293 CharacterOffsetEnd=298 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=298 CharacterOffsetEnd=299 PartOfSpeech=.] 
root(ROOT-0, live-3)
nsubj(live-3, I-1)
aux(live-3, can-2)
prep(live-3, with-4)
det(story-7, the-5)
amod(story-7, silly-6)
pobj(with-4, story-7)
punct(live-3, .-8)

Sentence #5 (40 tokens):
But when I see those dumbly grinning "main characters" with their pseudo-foreign speech (only Germans will understand what I talk about), I felt I kind of loathing I never thought I was capable of.
[Text=But CharacterOffsetBegin=304 CharacterOffsetEnd=307 PartOfSpeech=CC] [Text=when CharacterOffsetBegin=308 CharacterOffsetEnd=312 PartOfSpeech=WRB] [Text=I CharacterOffsetBegin=313 CharacterOffsetEnd=314 PartOfSpeech=PRP] [Text=see CharacterOffsetBegin=315 CharacterOffsetEnd=318 PartOfSpeech=VBP] [Text=those CharacterOffsetBegin=319 CharacterOffsetEnd=324 PartOfSpeech=DT] [Text=dumbly CharacterOffsetBegin=325 CharacterOffsetEnd=331 PartOfSpeech=RB] [Text=grinning CharacterOffsetBegin=332 CharacterOffsetEnd=340 PartOfSpeech=VBG] [Text=`` CharacterOffsetBegin=341 CharacterOffsetEnd=342 PartOfSpeech=``] [Text=main CharacterOffsetBegin=342 CharacterOffsetEnd=346 PartOfSpeech=JJ] [Text=characters CharacterOffsetBegin=347 CharacterOffsetEnd=357 PartOfSpeech=NNS] [Text='' CharacterOffsetBegin=357 CharacterOffsetEnd=358 PartOfSpeech=''] [Text=with CharacterOffsetBegin=359 CharacterOffsetEnd=363 PartOfSpeech=IN] [Text=their CharacterOffsetBegin=364 CharacterOffsetEnd=369 PartOfSpeech=PRP$] [Text=pseudo-foreign CharacterOffsetBegin=370 CharacterOffsetEnd=384 PartOfSpeech=JJ] [Text=speech CharacterOffsetBegin=385 CharacterOffsetEnd=391 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=392 CharacterOffsetEnd=393 PartOfSpeech=-LRB-] [Text=only CharacterOffsetBegin=393 CharacterOffsetEnd=397 PartOfSpeech=RB] [Text=Germans CharacterOffsetBegin=398 CharacterOffsetEnd=405 PartOfSpeech=NNPS] [Text=will CharacterOffsetBegin=406 CharacterOffsetEnd=410 PartOfSpeech=MD] [Text=understand CharacterOffsetBegin=411 CharacterOffsetEnd=421 PartOfSpeech=VB] [Text=what CharacterOffsetBegin=422 CharacterOffsetEnd=426 PartOfSpeech=WP] [Text=I CharacterOffsetBegin=427 CharacterOffsetEnd=428 PartOfSpeech=PRP] [Text=talk CharacterOffsetBegin=429 CharacterOffsetEnd=433 PartOfSpeech=VBP] [Text=about CharacterOffsetBegin=434 CharacterOffsetEnd=439 PartOfSpeech=IN] [Text=-RRB- CharacterOffsetBegin=439 CharacterOffsetEnd=440 PartOfSpeech=-RRB-] [Text=, CharacterOffsetBegin=440 CharacterOffsetEnd=441 PartOfSpeech=,] [Text=I CharacterOffsetBegin=442 CharacterOffsetEnd=443 PartOfSpeech=PRP] [Text=felt CharacterOffsetBegin=444 CharacterOffsetEnd=448 PartOfSpeech=VBD] [Text=I CharacterOffsetBegin=449 CharacterOffsetEnd=450 PartOfSpeech=PRP] [Text=kind CharacterOffsetBegin=451 CharacterOffsetEnd=455 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=456 CharacterOffsetEnd=458 PartOfSpeech=IN] [Text=loathing CharacterOffsetBegin=459 CharacterOffsetEnd=467 PartOfSpeech=NN] [Text=I CharacterOffsetBegin=468 CharacterOffsetEnd=469 PartOfSpeech=PRP] [Text=never CharacterOffsetBegin=470 CharacterOffsetEnd=475 PartOfSpeech=RB] [Text=thought CharacterOffsetBegin=476 CharacterOffsetEnd=483 PartOfSpeech=VBD] [Text=I CharacterOffsetBegin=484 CharacterOffsetEnd=485 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=486 CharacterOffsetEnd=489 PartOfSpeech=VBD] [Text=capable CharacterOffsetBegin=490 CharacterOffsetEnd=497 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=498 CharacterOffsetEnd=500 PartOfSpeech=IN] [Text=. CharacterOffsetBegin=500 CharacterOffsetEnd=501 PartOfSpeech=.] 
root(ROOT-0, But-1)
advmod(see-4, when-2)
nsubj(see-4, I-3)
dep(But-1, see-4)
det(characters-10, those-5)
amod(characters-10, dumbly-6)
amod(characters-10, grinning-7)
punct(characters-10, ``-8)
amod(characters-10, main-9)
dobj(see-4, characters-10)
punct(see-4, ''-11)
prep(see-4, with-12)
poss(speech-15, their-13)
amod(speech-15, pseudo-foreign-14)
pobj(with-12, speech-15)
punct(Germans-18, -LRB--16)
advmod(Germans-18, only-17)
nsubj(understand-20, Germans-18)
aux(understand-20, will-19)
rcmod(speech-15, understand-20)
dobj(talk-23, what-21)
nsubj(talk-23, I-22)
ccomp(understand-20, talk-23)
advmod(felt-28, about-24)
punct(felt-28, -RRB--25)
punct(felt-28, ,-26)
nsubj(felt-28, I-27)
ccomp(talk-23, felt-28)
nsubj(kind-30, I-29)
ccomp(felt-28, kind-30)
prep(kind-30, of-31)
pobj(of-31, loathing-32)
nsubj(thought-35, I-33)
neg(thought-35, never-34)
rcmod(kind-30, thought-35)
nsubj(capable-38, I-36)
cop(capable-38, was-37)
ccomp(thought-35, capable-38)
prep(capable-38, of-39)
punct(But-1, .-40)

Sentence #6 (18 tokens):
(If they had left them out, the thing may actually have been acceptable...).
[Text=-LRB- CharacterOffsetBegin=506 CharacterOffsetEnd=507 PartOfSpeech=-LRB-] [Text=If CharacterOffsetBegin=507 CharacterOffsetEnd=509 PartOfSpeech=IN] [Text=they CharacterOffsetBegin=510 CharacterOffsetEnd=514 PartOfSpeech=PRP] [Text=had CharacterOffsetBegin=515 CharacterOffsetEnd=518 PartOfSpeech=VBD] [Text=left CharacterOffsetBegin=519 CharacterOffsetEnd=523 PartOfSpeech=VBN] [Text=them CharacterOffsetBegin=524 CharacterOffsetEnd=528 PartOfSpeech=PRP] [Text=out CharacterOffsetBegin=529 CharacterOffsetEnd=532 PartOfSpeech=RP] [Text=, CharacterOffsetBegin=532 CharacterOffsetEnd=533 PartOfSpeech=,] [Text=the CharacterOffsetBegin=534 CharacterOffsetEnd=537 PartOfSpeech=DT] [Text=thing CharacterOffsetBegin=538 CharacterOffsetEnd=543 PartOfSpeech=NN] [Text=may CharacterOffsetBegin=544 CharacterOffsetEnd=547 PartOfSpeech=MD] [Text=actually CharacterOffsetBegin=548 CharacterOffsetEnd=556 PartOfSpeech=RB] [Text=have CharacterOffsetBegin=557 CharacterOffsetEnd=561 PartOfSpeech=VB] [Text=been CharacterOffsetBegin=562 CharacterOffsetEnd=566 PartOfSpeech=VBN] [Text=acceptable CharacterOffsetBegin=567 CharacterOffsetEnd=577 PartOfSpeech=JJ] [Text=... CharacterOffsetBegin=577 CharacterOffsetEnd=580 PartOfSpeech=:] [Text=-RRB- CharacterOffsetBegin=580 CharacterOffsetEnd=581 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=581 CharacterOffsetEnd=582 PartOfSpeech=.] 
root(ROOT-0, acceptable-15)
punct(acceptable-15, -LRB--1)
mark(left-5, If-2)
nsubj(left-5, they-3)
aux(left-5, had-4)
advcl(acceptable-15, left-5)
dobj(left-5, them-6)
prt(left-5, out-7)
punct(acceptable-15, ,-8)
det(thing-10, the-9)
nsubj(acceptable-15, thing-10)
aux(acceptable-15, may-11)
advmod(acceptable-15, actually-12)
aux(acceptable-15, have-13)
cop(acceptable-15, been-14)
punct(acceptable-15, ...-16)
punct(acceptable-15, -RRB--17)
punct(acceptable-15, .-18)

Sentence #7 (12 tokens):
There's not much more to be said about this one.
[Text=There CharacterOffsetBegin=582 CharacterOffsetEnd=587 PartOfSpeech=EX] [Text='s CharacterOffsetBegin=587 CharacterOffsetEnd=589 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=590 CharacterOffsetEnd=593 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=594 CharacterOffsetEnd=598 PartOfSpeech=RB] [Text=more CharacterOffsetBegin=599 CharacterOffsetEnd=603 PartOfSpeech=JJR] [Text=to CharacterOffsetBegin=604 CharacterOffsetEnd=606 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=607 CharacterOffsetEnd=609 PartOfSpeech=VB] [Text=said CharacterOffsetBegin=610 CharacterOffsetEnd=614 PartOfSpeech=VBN] [Text=about CharacterOffsetBegin=615 CharacterOffsetEnd=620 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=621 CharacterOffsetEnd=625 PartOfSpeech=DT] [Text=one CharacterOffsetBegin=626 CharacterOffsetEnd=629 PartOfSpeech=CD] [Text=. CharacterOffsetBegin=629 CharacterOffsetEnd=630 PartOfSpeech=.] 
root(ROOT-0, 's-2)
expl('s-2, There-1)
neg('s-2, not-3)
advmod(more-5, much-4)
advmod(said-8, more-5)
aux(said-8, to-6)
auxpass(said-8, be-7)
dep('s-2, said-8)
prep(said-8, about-9)
det(one-11, this-10)
pobj(about-9, one-11)
punct('s-2, .-12)

Sentence #8 (22 tokens):
You may laugh once or twice, because it's so ridiculous, but that doesn't make it any better.
[Text=You CharacterOffsetBegin=635 CharacterOffsetEnd=638 PartOfSpeech=PRP] [Text=may CharacterOffsetBegin=639 CharacterOffsetEnd=642 PartOfSpeech=MD] [Text=laugh CharacterOffsetBegin=643 CharacterOffsetEnd=648 PartOfSpeech=NN] [Text=once CharacterOffsetBegin=649 CharacterOffsetEnd=653 PartOfSpeech=RB] [Text=or CharacterOffsetBegin=654 CharacterOffsetEnd=656 PartOfSpeech=CC] [Text=twice CharacterOffsetBegin=657 CharacterOffsetEnd=662 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=662 CharacterOffsetEnd=663 PartOfSpeech=,] [Text=because CharacterOffsetBegin=664 CharacterOffsetEnd=671 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=672 CharacterOffsetEnd=674 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=674 CharacterOffsetEnd=676 PartOfSpeech=VBZ] [Text=so CharacterOffsetBegin=677 CharacterOffsetEnd=679 PartOfSpeech=RB] [Text=ridiculous CharacterOffsetBegin=680 CharacterOffsetEnd=690 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=690 CharacterOffsetEnd=691 PartOfSpeech=,] [Text=but CharacterOffsetBegin=692 CharacterOffsetEnd=695 PartOfSpeech=CC] [Text=that CharacterOffsetBegin=696 CharacterOffsetEnd=700 PartOfSpeech=IN] [Text=does CharacterOffsetBegin=701 CharacterOffsetEnd=705 PartOfSpeech=VBZ] [Text=n't CharacterOffsetBegin=705 CharacterOffsetEnd=708 PartOfSpeech=RB] [Text=make CharacterOffsetBegin=709 CharacterOffsetEnd=713 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=714 CharacterOffsetEnd=716 PartOfSpeech=PRP] [Text=any CharacterOffsetBegin=717 CharacterOffsetEnd=720 PartOfSpeech=DT] [Text=better CharacterOffsetBegin=721 CharacterOffsetEnd=727 PartOfSpeech=JJR] [Text=. CharacterOffsetBegin=727 CharacterOffsetEnd=728 PartOfSpeech=.] 
root(ROOT-0, laugh-3)
nsubj(laugh-3, You-1)
aux(laugh-3, may-2)
advmod(laugh-3, once-4)
cc(once-4, or-5)
conj(once-4, twice-6)
punct(laugh-3, ,-7)
mark(ridiculous-12, because-8)
nsubj(ridiculous-12, it-9)
cop(ridiculous-12, 's-10)
advmod(ridiculous-12, so-11)
advcl(laugh-3, ridiculous-12)
punct(laugh-3, ,-13)
cc(laugh-3, but-14)
nsubj(make-18, that-15)
aux(make-18, does-16)
neg(make-18, n't-17)
conj(laugh-3, make-18)
nsubj(better-21, it-19)
dep(better-21, any-20)
xcomp(make-18, better-21)
punct(laugh-3, .-22)

Sentence #9 (41 tokens):
It is definitely not funny.If you live outside of Germany, be happy and rejoice that this awful work will most likely never make it to your cinemas.If you do live in Germany, avoid this movie at all costs.
[Text=It CharacterOffsetBegin=733 CharacterOffsetEnd=735 PartOfSpeech=PRP] [Text=is CharacterOffsetBegin=736 CharacterOffsetEnd=738 PartOfSpeech=VBZ] [Text=definitely CharacterOffsetBegin=739 CharacterOffsetEnd=749 PartOfSpeech=RB] [Text=not CharacterOffsetBegin=750 CharacterOffsetEnd=753 PartOfSpeech=RB] [Text=funny.If CharacterOffsetBegin=754 CharacterOffsetEnd=762 PartOfSpeech=NN] [Text=you CharacterOffsetBegin=763 CharacterOffsetEnd=766 PartOfSpeech=PRP] [Text=live CharacterOffsetBegin=767 CharacterOffsetEnd=771 PartOfSpeech=VBP] [Text=outside CharacterOffsetBegin=772 CharacterOffsetEnd=779 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=780 CharacterOffsetEnd=782 PartOfSpeech=IN] [Text=Germany CharacterOffsetBegin=783 CharacterOffsetEnd=790 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=790 CharacterOffsetEnd=791 PartOfSpeech=,] [Text=be CharacterOffsetBegin=792 CharacterOffsetEnd=794 PartOfSpeech=VB] [Text=happy CharacterOffsetBegin=795 CharacterOffsetEnd=800 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=801 CharacterOffsetEnd=804 PartOfSpeech=CC] [Text=rejoice CharacterOffsetBegin=805 CharacterOffsetEnd=812 PartOfSpeech=VB] [Text=that CharacterOffsetBegin=813 CharacterOffsetEnd=817 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=818 CharacterOffsetEnd=822 PartOfSpeech=DT] [Text=awful CharacterOffsetBegin=823 CharacterOffsetEnd=828 PartOfSpeech=JJ] [Text=work CharacterOffsetBegin=829 CharacterOffsetEnd=833 PartOfSpeech=NN] [Text=will CharacterOffsetBegin=834 CharacterOffsetEnd=838 PartOfSpeech=MD] [Text=most CharacterOffsetBegin=839 CharacterOffsetEnd=843 PartOfSpeech=RBS] [Text=likely CharacterOffsetBegin=844 CharacterOffsetEnd=850 PartOfSpeech=JJ] [Text=never CharacterOffsetBegin=851 CharacterOffsetEnd=856 PartOfSpeech=RB] [Text=make CharacterOffsetBegin=857 CharacterOffsetEnd=861 PartOfSpeech=VBP] [Text=it CharacterOffsetBegin=862 CharacterOffsetEnd=864 PartOfSpeech=PRP] [Text=to CharacterOffsetBegin=865 CharacterOffsetEnd=867 PartOfSpeech=TO] [Text=your CharacterOffsetBegin=868 CharacterOffsetEnd=872 PartOfSpeech=PRP$] [Text=cinemas.If CharacterOffsetBegin=873 CharacterOffsetEnd=883 PartOfSpeech=NN] [Text=you CharacterOffsetBegin=884 CharacterOffsetEnd=887 PartOfSpeech=PRP] [Text=do CharacterOffsetBegin=888 CharacterOffsetEnd=890 PartOfSpeech=VBP] [Text=live CharacterOffsetBegin=891 CharacterOffsetEnd=895 PartOfSpeech=VB] [Text=in CharacterOffsetBegin=896 CharacterOffsetEnd=898 PartOfSpeech=IN] [Text=Germany CharacterOffsetBegin=899 CharacterOffsetEnd=906 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=906 CharacterOffsetEnd=907 PartOfSpeech=,] [Text=avoid CharacterOffsetBegin=908 CharacterOffsetEnd=913 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=914 CharacterOffsetEnd=918 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=919 CharacterOffsetEnd=924 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=925 CharacterOffsetEnd=927 PartOfSpeech=IN] [Text=all CharacterOffsetBegin=928 CharacterOffsetEnd=931 PartOfSpeech=DT] [Text=costs CharacterOffsetBegin=932 CharacterOffsetEnd=937 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=937 CharacterOffsetEnd=938 PartOfSpeech=.] 
root(ROOT-0, happy-13)
nsubj(funny.If-5, It-1)
cop(funny.If-5, is-2)
advmod(funny.If-5, definitely-3)
neg(funny.If-5, not-4)
nsubj(happy-13, funny.If-5)
nsubj(live-7, you-6)
rcmod(funny.If-5, live-7)
advmod(live-7, outside-8)
prep(outside-8, of-9)
pobj(of-9, Germany-10)
punct(funny.If-5, ,-11)
cop(happy-13, be-12)
cc(happy-13, and-14)
conj(happy-13, rejoice-15)
mark(make-24, that-16)
det(work-19, this-17)
amod(work-19, awful-18)
nsubj(make-24, work-19)
aux(make-24, will-20)
advmod(likely-22, most-21)
nsubj(make-24, likely-22)
neg(make-24, never-23)
ccomp(happy-13, make-24)
dobj(make-24, it-25)
prep(make-24, to-26)
poss(cinemas.If-28, your-27)
pobj(to-26, cinemas.If-28)
nsubj(live-31, you-29)
aux(live-31, do-30)
dep(make-24, live-31)
prep(live-31, in-32)
pobj(in-32, Germany-33)
punct(live-31, ,-34)
dep(live-31, avoid-35)
det(movie-37, this-36)
dobj(avoid-35, movie-37)
prep(avoid-35, at-38)
det(costs-40, all-39)
pobj(at-38, costs-40)
punct(happy-13, .-41)

